### PR TITLE
T 13113 shell open pr

### PR DESCRIPTION
**Pull request by dot2dotseurat**
PRs that come from forked repos do not run dbt cloud tests automatically. 

If we want to use the dbt slim CI job to check their work, we need an internal branch/PR. My recommendation is that we confirm nothing untoward is happening in the PR and if it looks good we run this shell script to create an internal PR for them to continue their work on.

Making an internal PR is sort of tedious so I wrote this script to make it easier for us. If we are happy with this flow, we can move it to a github action. 

n/a
I've checked that:

* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] the directory tree matches the pattern /sector/blockchain/ e.g. /tokens/ethereum
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each file has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`

### CODE REVIEW

**Comment by jeff-dude** [comment]
what about a step to close user-submitted PR (with automated comment/tag of new PR) after new one successfully opens?

**Comment by dot2dotseurat** [comment]
> what about a step to close user-submitted PR (with automated comment/tag of new PR) after new one successfully opens?

that *might* be possible. but I have no idea how to close a pull request from the command line.

**Comment by jeff-dude** [comment]
> > what about a step to close user-submitted PR (with automated comment/tag of new PR) after new one successfully opens?
> 
> that _might_ be possible. but I have no idea how to close a pull request from the command line.

no problem, something to keep in mind as we enhance 🙌

**Review by jeff-dude — APPROVED** [review]

File: git_scripts/open_internal_pr.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+### For Dune employees.
+### This script fetches a branch from a forked repo, opens a new internal branch, and opens a pull request
+### Example usage:
+### To open a pull request from 0xbitfly from their master branch
+### run the following command from your terminal in the abstractions repo
```
Comment (soispoke) [inline git_scripts/open_internal_pr.sh:6]: running it from the abstractions repo didn't work for me, I had to run it inside the git_scripts folder

**Review by soispoke — COMMENTED** [review]
Nice work ! We should test this w/ hildobby :)

**Review by soispoke — COMMENTED** [review]
I like @jeff-dude's idea, would https://cli.github.com/manual/gh_pr_close work for example ?

**Comment by dot2dotseurat** [comment]
> I like @jeff-dude's idea, would https://cli.github.com/manual/gh_pr_close work for example ?

this could work. we should test it with a remote branch.

**Pull request merged**